mono/mini/interp/interp.c

```diff
@@ -3664,6 +3677,29 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, FrameClause
 
 			MINT_IN_BREAK;
 		}
+		MINT_IN_CASE(MINT_JIT_CALL2) {
+#ifdef ENABLE_EXPERIMENT_TIERED
+			InterpMethod *rmethod = (InterpMethod *) READ64 (ip + 1);
+
+			MONO_API_ERROR_INIT (error);
```

Comment:
Should probably just be plain `error_init`?

Reply:
copy/paste from `MONO_JIT_CALL` (above)

Reply:
It should also probably change. Just because your neighbor does it, does not make it best.